docs/plugins/albumtypes.rst

```diff
@@ -0,0 +1,57 @@
+AlbumTypes Plugin
+=================
+
+The ``albumtypes`` plugin adds the ability to format and output album types,
+such as "Album", "EP", "Single", etc. List of available type can be found
+`here`_.
+
+To use the ``albumtypes`` plugin, enable it in your configuration
+(see :ref:`using-plugins`). Then, add ``$atypes`` to your path formats as
+desired.
+
+.. _here: https://musicbrainz.org/doc/Release_Group/Type
+
+Configuration
+-------------
+
+To configure the plugin, make a ``albumtypes:`` section in your configuration
+file. The available options are:
+
+- **types**: An ordered list of album type to format mappings. The order of the
+  mappings determines their order in the output. If a mapping is missing or
+  blank, it will not be in the output.
+  Default: ``[]``.
```

Comment:
Maybe just say "Default: None." (`[]` implies it's a list, not a mapping.)

But also… should the default be nothing? Does that mean that `$atypes` will always be empty by default? Perhaps this should come with a default mapping, like the one shown below, so the plugin is useful "out of the box"?

Reply:
That's a good idea, I think. I've set the default values to the example config and adjusted the docs accordingly.